addons/binding/org.openhab.binding.nest/README.md

```diff
@@ -0,0 +1,357 @@
+| timeout  | 10000   |   No     | HTTP request timeout in milliseconds |
+
+## Item configuration
+
+The syntax for the Nest item configuration string is explained below.
+
+Nest bindings start with a symbol character to indicate if the item receives values from the API (in binding `<`), sends values to the API (out binding `>`), or both (bidirectional binding `=`).
+
+The first character is then followed by a section between square brackets (\[and \] characters):
+
+```
+<[<property>]
+```
+
+where `<property>` is one of a long list of properties than you can read and optionally change. See the list below, and then read the [Nest API Reference](https://developer.nest.com/documentation/api-reference) for all specifics as to their meanings.
+
+Since device and structure identifiers are very long, cryptic strings that are hard to learn, binding configurations allow you to use the device's textual name as a reference.  Whatever name you see in the web or mobile client is the name you would supply in an Item's binding configuration.  So, for example, in order to determine the current humidity detected at the thermostat named 'Living Room', your binding configuration would look like this:
+
+```
+Number NestTStat_humidity "Humidity [%d %%]" {nest="<[thermostats(Living Room).humidity]"}
+```
+
+### Handling special characters
+
+With the convenience of using simple names for structures, thermostats and smoke+CO detectors, comes the price of having to handle special characters in the names.  Any characters in a name that could interfere with the parsing of the binding configuration string need to be either 1) removed from the device name in your account at nest.com, or 2) replaced with "URL-encoded" versions.  The characters that have to be replaced are `[`, `]`, `(`, `)`, `,`, `.` and `+`.  Here are some examples:
+
+What you see at nest.com | The escaped version to use in the Nest binding
+-------------------------------------|-----------------------------------------------
+Dining Room (Ground Floor) | Dining Room %28Ground Floor%29
+Den Smoke+CO | Den Smoke%2BCO
+123 Main St. | 123 Main St%2E
+Bogota, Colombia | Bogota%2C Colombia
+[Basement] | %5BBasement%5D
+
+To reiterate, you could change the display names of your devices at nest.com, and thereby avoid having to put escaped versions in your binding configuration strings.
+
+In order to change the current HVAC mode of the Living room thermostat between `off`, `eco`, `heat`, `cool` and `heat-cool`, your item would look like:
+
+```
+String NestTStat_hvac_mode "HVAC Mode [%s]" {nest="=[thermostats(Living Room).hvac_mode]"}
```

Comment:
This documentation seems to be outdated. Shouldn't we use 
```
{ channel="nest.thermomstat.LivingRoom.hvac_mode" }
```
syntax, or something like that?